pseudocode.js

```diff
@@ -66,4 +66,10 @@ module.exports = {
             }
         }
     },
+
+    renderClass: function(elements, options) {
+        for (let index = elements.length - 1; index >= 0; index--) {
+            this.renderElement(elements[index], options);
+        }
+    },
```

Comment:
I think this function could be improved. It's a `renderClass` function, but it accepts an array of elements, rather than a class name.

How about:
```suggestion
    renderClass: function(class_name, options) {
      [].forEach.call(
        document.getElementsByClassName(class_name),
        function (el) { this.renderElement(el, options); }
      );
    },
```

And then you could use it as:
```js
pseudocode.renderClass("pseudocode",
                       {
                           lineNumber: false,
                           noEnd: true
                       });
```